RELEASING.md

```diff
@@ -120,3 +123,45 @@ github repository.
     $ git push upstream v$MAJOR.$MINOR.$PATCH
     $ git push upstream v$MAJOR.$MINOR.x
     ```
+
+## Deployment
+
+Deployment to Maven Central (or the snapshot repo) is for all of the artifacts
+from the project.
+
+### Branch
+
+Before building/deploying, be sure to switch to the appropriate branch or tag.
+For the current release use:
+
+```bash
+$ git checkout -b v$MAJOR.$MINOR.$PATCH
```

Comment:
So these two commands require you to have a remote branch `upstream/v$MAJOR.$MINOR.$PATCH`?  Could you use `git checkout -b v$MAJOR.$MINOR.$PATCH tags/v$MAJOR.$MINOR.$PATCH` so that only the tag is required?  The command only succeeds if the tag exists, so it is less likely to check out the wrong commit.